### DISCUSSION TITLE
Not all tailwind classes work?

### DISCUSSION THREAD

**Discussion by ariahrism** [zauberzeug/nicegui#2343] [Q&A] [answered]
### Question

```py
from nicegui import ui

ui.button('', icon='search').classes('min-h-14').style('')
ui.button('', icon='search').classes('min-h-[3.5rem]').style('')
ui.button('', icon='search').classes('').style('min-height: 3.5rem;')

ui.run()
```

The first line doesn't work, but these three buttons, as per the documentation, should look the same:
![image](https://github.com/zauberzeug/nicegui/assets/19563239/19289a68-fe2f-4b9d-9aaa-357b0d8717b6)

Am I doing something wrong, or?

This is just one concrete example, but I have quite a few others where I felt I needed to resort to `.style()` because tailwind didn't work. Another being: min-h-svh  (min-height: 100svh; works)

**Comment by falkoschindler** [comment, marked as answer]
Hi @ariahrism!

It looks like the class "min-h-14" has been added in [Tailwind 3.4.0](https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.4.0):
> Add spacing scale to min-w-*, min-h-*, and max-w-* utilities (https://github.com/tailwindlabs/tailwindcss/pull/12300)

NiceGUI is currently using [Tailwind 3.2.0](https://github.com/iron3oxide/nicegui/blob/04e9cb13caee3362c7dea03426a072ec27f03a98/DEPENDENCIES.md#L5). We will upgrade to a newer Tailwind version at the latest with the next major release.

**Reply by ariahrism** [reply]
Ok, I had a suspicion that might be the case, but I wasn't sure how to confirm.
Thank you for taking the time to confirm the source of the issue!
